### DISCUSSION TITLE
"What's this bench made of?" quest doubts

### DISCUSSION THREAD

**Discussion by ivanbranco** [Helium314/SCEE#435] [General]
Hi, I have some doubts about this quest. Should I indicate the material of the base/frame, the material of the seating, or should I reply only when base/frame and seating are made of the same material? Since this is not strictly related to SCEE but to benches tagging in general, I opened a discussion in the Community Forum if you are interested: https://community.openstreetmap.org/t/benches-material/100491

**Comment by Helium314** [comment]
Sorry for the late reply, for some reason I had turned off notifications about new discussions.

Documentiation on [Wiki](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbench) is poor, e.g. the 5th example has `material=concrete`, `surface=wood`, while the 3rd example has `material=wood` without caring about the bricks below.

In the discussion people don't agree anyway, and the actual usage for `material` seems to mostly refer to the seating instead of what is below. If you further continue some of the reasoning in the discussion, you can argue that actually you should tag surface=paint/varnish/whatever for many benches. Maybe technically correct and a valid interpretation of "_surface=* – The surface of the seating if different from material=*_", but likely not what people would expect.

Personally I go with the common usage and select the seating material.
